ml/libs/decode_audio.py

```
@@ -0,0 +1,30 @@
+        "ffmpeg",
+        "-nostdin",
+        "-hide_banner",
+        "-loglevel", "error",
+        "-i", "pipe:0",
+        "-f", "f32le",
+        "-ac", "1",
+        "-ar", str(sr),
+        "pipe:1",
+    ]
+    try:
+        proc = subprocess.run(cmd, input=data, capture_output=True)
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)